### PR TITLE
add recipe 11.2

### PR DESCRIPTION
**Pull request by ericbjones**
First pr to this project so just opening it for discussion, if there are any unit tests I am supposed to add or such I will get to doing so soon.  Thanks

### CODE REVIEW

**Comment by adrianyorke** [comment]
Looks good, thanks for the pr.  All tests passing.  You could also update README.md at the top level and add a link for the new recipe.  I can also do this if you prefer, let me know.

**Comment by adrianyorke** [comment]
Add to *** Settings *** section:

Force Tags       py3.6    py3.7    py3.8

**Comment by adrianyorke** [comment]
@ericbjones: did you want to push more commits?  I can merge if you wish to finalise this PR.

**Comment by ericbjones** [comment]
@adrianyorke if you do not mind adding the minor fixes, that would be great! :)

File: src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+Run Remote ${Command} And Get ${File} Output
+    [Documentation]          This keyword uses a variable keyword name, meaning that it can be called by any name which matches
+    ...                      the name defined, but with any string in place of the variables. It is one of the more extravigant
+    ...                      feautures of robot and may feel too quirky to some python developers. But depending on style choices
+    ...                      for your project, you may choose to use this feature.
+    ...                    
+    ...                      To the point, this keyword runs the command and outputs to a file on the server, then downloads that file
+    ...                      and prints the content to console. Replace printing to cosole with whatever you actually need to do with
```
Comment (ericbjones) [inline src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot:63]:
```suggestion
    ...                      and returns the local file location so you can print to console or do whatever you need to do with
```

File: src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot [hunk 2]
```diff
@@ -0,0 +1,78 @@
+*** Settings ***
+Documentation                PROBLEM:
+...                          You need to execute a command on a remote machine via SSH then get its vary large text output, or some
+...                          bianary artifact that the remote command creates using file transfer.
+...                          DISCUSSION:
+...                          This recipe demonstrates using keywords from SSH external library
+...                          combined with standard library OperatingSystem. In this recipie we are
+...                          loging in with ssh public key to demonstate that option for logins.
```
Comment (ericbjones) [inline src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot:8]:
```suggestion
...                          logging in with ssh public key to demonstrate that option for logins.
```

File: src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+...                          SEE ALSO:
+...                          SSHLibrary documentation: http://robotframework.org/SSHLibrary/SSHLibrary.html
+                         
+Library                      SSHLibrary         WITH NAME    SSH
+Library                      OperatingSystem    WITH NAME    OS
+                         
+Suite Setup                  Open Connection And Log In
+Suite Teardown               SSH.Close All Connections
+Force Tags                   no-ci-testing
+
```
Comment (ericbjones) [inline src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot:26]:
```suggestion
Force Tags                    py3.6 py3.7 py3.8

```
@adrianyorke is that right?

File: src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+...                          $ pip install --upgrade robot framework-sshlibrary
+...                          SEE ALSO:
+...                          SSHLibrary documentation: http://robotframework.org/SSHLibrary/SSHLibrary.html
+                         
+Library                      SSHLibrary         WITH NAME    SSH
+Library                      OperatingSystem    WITH NAME    OS
+                         
+Suite Setup                  Open Connection And Log In
+Suite Teardown               SSH.Close All Connections
+Force Tags                   no-ci-testing
```
Comment (ericbjones) [inline src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot:25]:
```suggestion
Force Tags                   no-ci-testing    py3.6     py3.7     py3.8
```
@adrianyorke is this correct?

Comment (adrianyorke) [reply src/section_11/recipe112_retrieve_output_file_of_ssh_run_command.robot:25]: Yes, that's great thanks.

**Comment by adrianyorke** [comment]
Merged - thanks for the contribution @ericbjones!

**Comment by adrianyorke** [comment]
@all-contributors please add @ericbjones for Code, Documentation

**Pull request merged**